translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md)  [:bosnia_herzegovina: Bosnian](README.bosnian.md) |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md)  [:bosnia_herzegovina: Bosnian](README.bosnian.md) | [:cn: 中文](README.chinese.md) |
```

Comment:
The Chinese translation doesn't show here:

![image](https://user-images.githubusercontent.com/5722022/71749721-888a3900-2e76-11ea-9fa6-709214b2b1fc.png)